tests/jax/ops/test_conv1d.py

```diff
@@ -0,0 +1,47 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import jax
+import pytest
+from infra import ComparisonConfig, random_tensor, run_op_test
+
+
+@pytest.fixture
+def comparison_config() -> ComparisonConfig:
+    config = ComparisonConfig()
+    config.disable_all()
+    config.pcc.enable()
+    config.pcc.required_pcc = 0.95
+    return config
```

Comment:
Please add some TODO to investigate why `conv1d` has such poor precision. Same in `conv2d` test.